src/openrct2/platform/Platform.Win32.cpp

```diff
@@ -37,6 +37,10 @@
 
 #    include <iterator>
 
+#    if _WIN32_WINNT < 0x600
+#        define swprintf_s(a, b, c, d, ...) swprintf(a, b, c, ##__VA_ARGS__)
```

Comment:
This drops the fourth argument, doesn't that mean that where this is used it prints garbage now?

Reply:
The fourth argument is buffer length, unused in the non-`_s` version.

Reply:
I would rather like to see this not being a macro, you can use variadic template arguments and perfect forwarding.

Reply:
That would just shift the ifdef into a template, is that what you mean?

Reply:
Just the function you still need the if _WIN32_WINNT < 0x600